code/modules/modular_computers/file_system/programs/research/passive_pointes.dm

```diff
@@ -45,8 +45,11 @@
 
 	if(progress >= target_progress)
 		reset()
+		computer.visible_message(SPAN_NOTICE("[computer] state, \"Decryption Succesfully Completed.\""))
+		playsound(computer.loc, 'sound/machines/ping.ogg', 50, 1 -3)
 		message = "Successfully collected data points and saved metadata results."
-		HD.store_file(new/datum/computer_file/binary/research_points())
+		var/datum/computer_file/binary/research_points/RP = new(target_progress/1000) // 1 Size = 1000 points.
```

Comment:
This makes it 1 points rather then 1000

Reply:
No, because 1 size of a data file = 1000 RnD point.

Reply:
This was resolved im dumb